.gitlab-ci.yml

```diff
@@ -12,5 +17,16 @@ include:
       - '.gitlab-ci-check-commits.yml'
       - '.gitlab-ci-check-license.yml'
 
-# Keep golang version aligned with latest yocto release
-image: golang:1.17-alpine3.16
+trigger:mender-qa:
+  stage: trigger
+  rules:
+    - if: '$CI_COMMIT_BRANCH =~ /^(master|[0-9]+\.[0-9]+\.x)$/'
+    # the following is to prevent an endless loop of qa pipelines caused by downstream pipelines
+    - if: '$CI_PIPELINE_SOURCE == "pipeline"'
+      when: never
+    - if: '$CI_PIPELINE_SOURCE == "schedule"'
+      when: never
+  trigger:
+    include:
+      - artifact: gitlab-ci-client-qemu-publish-job.yml
+        job: generate-qa-trigger
```

Comment:
We can remove the whole trigger, as I said in the previous round it is an overkill. Do you disagree?

IOW I don't expect changes in `.gitlab-ci.yml` from this PR.

Reply:
Roger :vulcan_salute: